test/integration_test.js

```diff
@@ -22,7 +22,11 @@ describe("Integration tests", () => {
   var sandbox, server, tasks;
   const MAX_ATTEMPTS = 50;
 
-  function startServer(env={}) {
+  function startServer(env) {
+    // Add the provided environment variables to the child process environment.
+    // Keeping parent's environment is needed so that pserve's executable 
+    // can be found (with PATH) if KINTO_PSERVE_EXECUTABLE env variable was not provided.
```

Comment:
Well, it's actually useful if you *activate* your venv. Otherwise the pserve executable needs to be explicitely called out to be found.